lib/fakezoomapi.ts

```diff
@@ -1,10 +1,24 @@
-import { DrawImageCallback, ZoomApiWrapper } from "./zoomapi";
+import {
+  DrawImageCallback,
+  ZoomApiWrapper,
+  AuthorizeCallback,
+  AuthorizeOptions,
+  GeneralMessageResponse,
+} from "./zoomapi";
 
 class FakeZoomApi implements ZoomApiWrapper {
   async setDrawImageCallback(cb: DrawImageCallback): Promise<void> {}
+  async authorize(data: AuthorizeOptions): Promise<GeneralMessageResponse> {
+    return { message: "Success" };
+  }
+  async setAuthorizeCallback(cb: AuthorizeCallback): Promise<void> {
+    setTimeout(() => cb({ code: "mocked_code" }), 100);
```

Comment:
Can we get rid of the delay here?
```suggestion
    setTimeout(() => cb({ code: "mocked_code" }), 0);
```

If not, can you add a comment about what it breaks?

Reply:
yes, remove setTimeout just call cb is ok